spec/sidekiq-instrument/server_middleware_spec.rb

```diff
@@ -3,13 +3,20 @@
 RSpec.describe Sidekiq::Instrument::ServerMiddleware do
   describe '#call' do
     let(:expected_dog_options) { { tags: ['queue:default', 'worker:my_worker'] } }
+    let(:worker_metric_name) do
+      "sidekiq_instrument_trace_workers::in_queue"
```

Comment:
Nit: Quote style consistency:
```suggestion
      'sidekiq_instrument_trace_workers::in_queue'
```